### DISCUSSION TITLE
How do I Add NPM Package to my git repository

### DISCUSSION THREAD

**Discussion by enjoyincity** [npm/feedback#444] [Website] [answered]
Hello guys!

I am a begginer website designing who basically wants to remove all bugs related to onpage seo from my website https://www.enjoyincity.in so that i can get good ranking in google on **mumbai russian escorts**  and other escort based keywords that cetegory does not have any specific schema and can not be submitted in many good do follow websites. So it is required to remove all onpage seo related bugs. for that i want to use github and try adding npm packages. 

I dont know about git and npm much. kindly help me how do i add npm package to remove all errors given by lighthouse. Likewise i want to put cache expiring timing to all static assets on my website mentioned above. I want to use minified and combined css and js resources. i want to inline all important css and js code. 

I am using windows 10 and here you can find my current github respositories [https://github.com/enjoyincity/](https://github.com/enjoyincity/). 

Please guys help me out. I am not a web developer and can not buy any paid package as of now.

Thanks

**Comment by MylesBorins** [comment, marked as answer]
Hey @enjoyincity. This discussions form is for suggesting features for the npm registry, website, and CLI. We do not offer general technical support through this channel. You may want to try opening an issue over at https://github.community or on Stackoverflow. Best of luck!
